lib/providers/snackbar/snackbarContext.tsx

```diff
@@ -0,0 +1,11 @@
+// SnackbarContext.js
```

Comment:
let's rename the file to `snackbar-content.tsx`

Reply:
you want to change context and provider files to `snackbar-context.tsx` and `snackbar-provider.tsx`?